README.adoc

```diff
@@ -444,6 +444,25 @@ Name of the docker image to build
 buildDockerImage('plugins-site-api')
 ----
 
+=== buildDockerImage_k8s
+
+Builds a standard docker image and on master/tags pushes it to dockerhub but runs on the k8s servers (infra, releases, etc)
```

Comment:
Reference https://github.com/jenkinsci/jep/blob/master/jep/217/README.adoc and other architecture aspects under the hood_

Reply:
I don't understand what this comment means